### DISCUSSION TITLE
Pimcore 11 Alpha Releases! 🧑‍🚀🚀

### DISCUSSION THREAD

**Discussion by dvesh3** [pimcore/pimcore#13841] [General]
## UPDATE: Pimcore 11 ALPHA7 Release 
Hey! We just tagged another Alpha release. Below are the link to the changes: 
https://github.com/pimcore/pimcore/tree/v11.0.0-ALPHA7/doc/23_Installation_and_Upgrade/09_Upgrade_Notes

## UPDATE: Pimcore 11 ALPHA5 Release
Hey! we just tagged another Alpha release. Below are some of the important changes:
 - [Security] use symfony native way of authentication and session management https://github.com/pimcore/pimcore/pull/13909
 - Replace SensioFrameworkExtraBundle with PHP 8 attribute https://github.com/pimcore/pimcore/pull/14062
 - Move Application Logger to ApplicationLoggerBundle within bundles folder in pimcore/pimcore repository https://github.com/pimcore/pimcore/pull/13930
 - Made `symfony/dotenv` package optional https://github.com/pimcore/pimcore/pull/14117
 - Cancel batch export https://github.com/pimcore/pimcore/pull/14173
 - Add config to register translations domains as valid https://github.com/pimcore/pimcore/pull/14091
and more..
[11.0.0-ALPHA5](https://github.com/pimcore/pimcore/releases/tag/v11.0.0-ALPHA5).
## UPDATE: Pimcore 11 ALPHA3 Release
Hello Pimconauts,

what better way to end a workday than with a pimcore alpha release? 

Checkout some of the new improvements and fixes that made it into the release: 

- Following functionality was extracted into a separate bundle (in pimcore/bundles):
  - Seo
  - SystemInfo
  - UUID
  - XLIFF
  - Word export
  - more to come
- [Improvement]: JS Events cancelable https://github.com/pimcore/pimcore/pull/13955
- [DataObjects] Consistent way of storing null/empty textfields to the database in https://github.com/pimcore/pimcore/pull/13963
- ... and many more

[11.0.0-ALPHA3](https://github.com/pimcore/pimcore/releases/tag/v11.0.0-ALPHA3).

Have a nice day and be safe in space.

## UPDATE: Pimcore 11 ALPHA2 Release
Hello Pimconauts,

we just tagged another Alpha release and special thanks to our community for their contributions 💜 

Below are some of the important changes including various fixes and improvements:
- Added required `ClassSavedInterface` for Data types implementing `classSaved` method https://github.com/pimcore/pimcore/pull/13846
- Fixed custom firewall logout deletes complete session - https://github.com/pimcore/pimcore/pull/13825
- Improved Tags assignment UI - https://github.com/pimcore/pimcore/pull/13844
- Glossary core feature now moved to its own Bundle https://github.com/pimcore/pimcore/pull/13811
- Removed Adminer Database Explorer https://github.com/pimcore/pimcore/pull/13921

You'll find more details in the tag [11.0.0-ALPHA2](https://github.com/pimcore/pimcore/releases/tag/v11.0.0-ALPHA2).

## Pimcore 11 ALPHA1 ready

Hi all, 
we are in the middle of our space journey to Pimcore 11. As for now, the biggest breaking changes should be done and this is why we tagged the first alpha release of Pimcore 11 🥳. 

You'll find the tag here [11.0.0-ALPHA1](https://github.com/pimcore/pimcore/releases/tag/v11.0.0-ALPHA1) and the upgrade notes in the [docs](https://github.com/pimcore/pimcore/tree/baa986dba542e6eb6145ab049fe71ceeaad0cc3f/doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes#1100).

**Keep in mind - it is an alpha, so there is still a lot to do!**

But it is also a good time for you, our beloved community, to kick in and give us some hands. 
Please play around, tell us what you think, help us testing and improving by providing PRs and feedback. We are looking forward to a lot contributions. 

### How to install?
you can install the Alpha version of skeleton or demo with docker command:
 - **skeleton**: ``docker run -u `id -u`:`id -g` --rm -v `pwd`:/var/www/html pimcore/pimcore:php8.1-latest composer create-project pimcore/skeleton:11.0.0-ALPHA1 my-project`` and follow rest of the steps [here](https://github.com/pimcore/demo#follow-these-steps).
 
 - **demo**: ``docker run -u `id -u`:`id -g` --rm -v `pwd`:/var/www/html pimcore/pimcore:php8.1-latest composer create-project pimcore/demo:11.0.0-ALPHA1 my-project`` and follow rest of the steps [here](https://github.com/pimcore/demo#follow-these-steps).

In the meantime, for all who are celebrating, have some nice holidays and all the best for the next year🎄🎊🧨🎆.

**Comment by Cruiser13** [comment]
Congratulations! Just a minor thought here - it would be great if alpha, beta and RC versions are not showing up as new versions to update to in productive pimcore environments:
![pimcore-11-alpha-update-hint](https://user-images.githubusercontent.com/2771909/209370249-912d8103-e5af-42e4-8146-e477bfe714fa.png)
This will save some hours discussing with clients not knowing what "alpha" means and demanding an update "as soon as possible" before the holidays.

All the best for the next year and the further pimcore 11 development, we're excited!

**Reply by mattamon** [reply]
@Cruiser13 thank you! :) 
You are totally right! I created a new issue for this! 

Wish you all the best for 2023!

**Reply by brusch** [reply]
@Cruiser13 fixed, thanks for reporting 😉
